BioCro/DARPA/biocro_biomass_clean_darpa.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Setaria Biomass Data Cleaning & Visualization"
+author: "Kristina Riemer, University of Arizona"
+output: github_document
+urlcolor: blue
+---
+
+### Overall summary
+
+We can definitely use one experiment's data (the second one), which we have already started to do. We might be able to use experiments 4 and 5, but they only have two harvest dates (do we need six harvest dates?). Some experiments (1, 3, 6) have only one harvest date for each unique treatment. And the last experiment definitely cannot be used currently because there's no dry biomass measurements. 
```

Comment:
Reword to identify the actual experiment used: something like "We can use experiment 2's data, which we ..."? 
May also want to remove, reword second sentence? It looks like a note to someone